daemon/daemon.go

```diff
@@ -1157,14 +1190,121 @@ func NewDaemon(ctx context.Context, config *config.Config, pluginStore *plugin.S
 			}
 		}
 
-		// TODO: imageStore, distributionMetadataStore, and ReferenceStore are only
-		// used above to run migration. They could be initialized in ImageService
-		// if migration is called from daemon/images. layerStore might move as well.
-		d.imageService = images.NewImageService(imgSvcConfig)
+		// If no containers are running, check whether can migrate image service
```

Comment:
```suggestion
		// If no containers are present, check whether can migrate image service
```

This shouldn't happen if there are container created, but are not running right?

Reply:
Updated, the status of the container should not make a difference for this check